changelog.d/PR342.bugfix

```diff
@@ -0,0 +1 @@
+Handle CORS for GetValidated3pidServlet. Endpoint /3pid/getValidated3pid returns valid CORS headers.
```

Comment:
Additionally, this file needs to be named `342.bugfix` instead of `PR342.bugfix`. Otherwise `towncrier` will attempt to link to `https://github.com/matrix-org/sydent/issues/PR342`, which doesn't work.

Reply:
Renamed